src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,88 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private int size;
+    private Node<K, V> [] table;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        int threshold = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
+        Node<K, V> node = new Node<>(key, value, null);
+        int indexOfTable = hash(key);
+        if (size == threshold) {
+            resize();
+        }
```

Comment:
1. I'd move part of logic from here to your resize method in order to make public method shorter and cleaner
2. Don't like name hash for method, it doesn't return hash itself
```suggestion
        resizeIfRequired();
        Node<K, V> node = new Node<>(key, value, null);
        int indexOfTable = getIndexByHash(key);
```